resultsdbupdater/utils.py

```diff
@@ -236,25 +227,49 @@ def handle_ci_umb(msg):
     if isinstance(system, list):
         system = system[0] if system else {}
 
-    result_data = {
-        'item': item,
-        'type': item_type,
-        'brew_task_id': brew_task_id,
-        'category': msg_body['category'],
-        'component': component,
-        'scratch': scratch,
-        'issuer': msg_body_artifact.get('issuer'),
-        'rebuild': msg_body['run'].get('rebuild'),
-        'log': msg_body['run']['log'],  # required
-        'system_os': system.get('os'),
-        'system_provider': system.get('provider'),
-        'ci_name': msg_body_ci.get('name'),
-        'ci_url': msg_body_ci.get('url'),
-        'ci_environment': msg_body_ci.get('environment'),
-        'ci_team': msg_body_ci.get('team'),
-        'ci_irc': msg_body_ci.get('irc'),
-        'ci_email': msg_body_ci.get('email'),
-    }
+    if item_type == "compose":
+        result_data = {
```

Comment:
Optional: You could define `result_data` outside the if statement with the common key value pairs, and then update the dict in the if statements

Reply:
Tried that but it looks bit easier to read and change this way.